README.rst

```diff
@@ -97,9 +97,16 @@ Running tests
 
 SageMaker Python SDK has unit tests and integration tests.
 
+You can install the libraries needed to run the tests by running :code:`pip install --upgrade .[test]` or, for Zsh users: :code:`pip install --upgrade .\[test\]`
+
 **Unit tests**
 
-tox is a prerequisite for running unit tests so you need to make sure you have it installed. To run the unit tests:
+
+We run unit tests with tox, which is a program that lets you run unit tests for multiple Python versions, and also make sure the
+code fits our style guidelines. We run tox with Python 2.7 and 3.5, so to run unit tests
+with the same configuration we do, you'll need to have interpreters for Python 2.7 and Python 3.5 installed.
```

Comment:
because #679 got merged, can you change 3.5 to 3.6?

Reply:
nice, done.